src/Renderer/OrientedImageCamera.js

```diff
@@ -0,0 +1,60 @@
+import * as THREE from 'three';
+
+class OrientedImageCamera extends THREE.PerspectiveCamera {
+    constructor(size = 1024, focal = 1024, center, near = 0.1, far = 1000, skew, aspect) {
+        size = size.isVector2 ? size : new THREE.Vector2(size);
+        aspect = aspect || size.x / size.y;
+        super(undefined, aspect, near, far);
+        this.size = size;
+        this.focal = focal.isVector2 ? focal : new THREE.Vector2(focal, focal);
+        this.center = center || size.clone().multiplyScalar(0.5);
+        this.skew = skew || 0;
+        Object.defineProperty(this, 'fov', {
+            get: () => Math.atan2(this.size.y, 2 * this.focal.y) * 360 / Math.PI,
```

Comment:
This value should be computed somewhere else, not every time we ask for the fov.

Reply:
This class makes `fov` a computed property depending on size.y and focal. An alternative would be to use setters for size and focal that update a cached fov value.